### PR TITLE
refactor: removed the unused constructor

### PR DESCRIPTION
**Pull request by alkavats1**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: devtools/projects/ng-devtools-backend/src/lib/hooks/identity-tracker.ts [hunk 1]
```diff
@@ -33,8 +33,6 @@ export class IdentityTracker {
   private _currentDirectiveId = new Map<any, number>();
   isComponent = new Map<any, boolean>();
 
-  // private constructor for Singleton Pattern
```
Comment (JeanMeche) [inline devtools/projects/ng-devtools-backend/src/lib/hooks/identity-tracker.ts:36]: This empty constructor serves a purpose as it is private. Please revert this change.

Comment (alkavats1) [reply devtools/projects/ng-devtools-backend/src/lib/hooks/identity-tracker.ts]: done

**Review by alxhub — APPROVED** [review]

**Comment by pkozlowski-opensource** [comment]
@alkavats1 this commit has the "docs:" prefix in the commit message but it changes the code: Please change the commit message prefix to `refactor: `

**Comment by alkavats1** [comment]
> @alkavats1 this commit has the "docs:" prefix in the commit message but it changes the code: Please change the commit message prefix to `refactor: `

Hi @pkozlowski-opensource I have refactored the commit message, please review

**Comment by pkozlowski-opensource** [comment]
@alkavats1 the lint check is still failing with "The commit message body does not meet the minimum length of 20 characters". You can always check the status of the lint checks by inspecting the outcome of the CI jobs.

**Comment by alkavats1** [comment]
> @alkavats1 the lint check is still failing with "The commit message body does not meet the minimum length of 20 characters". You can always check the status of the lint checks by inspecting the outcome of the CI jobs.

Hi @pkozlowski-opensource I have resolved that linting issue, can you please review it now?
Thanks

**Comment by pkozlowski-opensource** [comment]
This PR was merged into the repository by commit afb7ea4e8cd5e7e0315ed342e5c9f446cfca6969.
